<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NLPMainWindow</class>
 <widget class="QMainWindow" name="NLPMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>자연어 처리 툴</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../common/resources/app_icon.ico</normaloff>../common/resources/app_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="rawdata_tblview"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일(&amp;F)</string>
    </property>
    <addaction name="menu_action_open"/>
    <addaction name="separator"/>
    <addaction name="menu_action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_visual">
    <property name="title">
     <string>시각화(&amp;V)</string>
    </property>
    <addaction name="menu_action_wordcloud"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>도움말(&amp;H)</string>
    </property>
    <addaction name="menu_action_help"/>
    <addaction name="menu_action_license"/>
    <addaction name="separator"/>
    <addaction name="menu_action_information"/>
   </widget>
   <widget class="QMenu" name="menu_analysis">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>분석(&amp;A)</string>
    </property>
    <addaction name="menu_action_wordcount"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_analysis"/>
   <addaction name="menu_visual"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="menu_action_open">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/HANENCIA/.designer/backup/common/resources/file_open.png</normaloff>C:/Users/HANENCIA/.designer/backup/common/resources/file_open.png</iconset>
   </property>
   <property name="text">
    <string>열기(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menu_action_wordcloud">
   <property name="text">
    <string>워드 클라우드(&amp;W)</string>
   </property>
  </action>
  <action name="menu_action_help">
   <property name="text">
    <string>도움말(&amp;H)</string>
   </property>
  </action>
  <action name="menu_action_license">
   <property name="text">
    <string>라이센스 정보</string>
   </property>
  </action>
  <action name="menu_action_quit">
   <property name="text">
    <string>종료(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="menu_action_information">
   <property name="text">
    <string>정보</string>
   </property>
  </action>
  <action name="menu_action_wordcount">
   <property name="text">
    <string>단어 빈도 분석</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
